doc/source/examples/networktheory/CPW media example.ipynb

```diff
@@ -0,0 +1,269 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Looking at measurements\n",
+    "\n",
+    "We look at measurement insertion loss phase and magnitude as well as return loss phase and magnitude.\n",
+    "\n",
+    "The insertion loss of CPWG200 is twice that of CPWG100 and is mostly linear up to 6 GHz. The phase shows also a two factor. Both statements agree with the fact CPWG200 has twice the length of CPWG100.\n",
+    "\n",
+    "The return loss insertion and phase have the same order of magnitude for the two networks. Insertion loss is below -20 dB at low frequency which indicate a good match of the impedance. The distance between magnitude minima is two time shorter with CPWG200 related to CPWG100, which again agrees with the physical length difference. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load raw measurements\n",
+    "TL100 = rf.Network('CPWG100.s2p')\n",
+    "TL200 = rf.Network('CPWG200.s2p')\n",
+    "\n",
+    "# plot them all\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.suptitle('Raw measurements')\n",
+    "plt.subplot(2, 2, 1)\n",
+    "TL100.plot_s_db(0, 0)\n",
+    "TL200.plot_s_db(0, 0)\n",
+    "TL100.plot_s_db(1, 1)\n",
+    "TL200.plot_s_db(1, 1)\n",
+    "plt.subplot(2, 2, 2)\n",
+    "TL100.plot_s_deg(0, 0)\n",
+    "TL200.plot_s_deg(0, 0)\n",
+    "TL100.plot_s_deg(1, 1)\n",
+    "TL200.plot_s_deg(1, 1)\n",
+    "plt.subplot(2, 2, 3)\n",
+    "TL100.plot_s_db(1, 0)\n",
+    "TL200.plot_s_db(1, 0)\n",
+    "TL100.plot_s_db(0, 1)\n",
+    "TL200.plot_s_db(0, 1)\n",
+    "plt.subplot(2, 2, 4)\n",
+    "TL100.plot_s_deg(1, 0)\n",
+    "TL200.plot_s_deg(1, 0)\n",
+    "TL100.plot_s_deg(0, 1)\n",
+    "TL200.plot_s_deg(0, 1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Removing connectors effects\n",
+    "\n",
+    "The SMA connectors cause a discontinuity in the signal path. The waves have to go trough a coaxial to planar transition with a complex three dimensionnal geometry in the launch region.\n",
+    "\n",
+    "By splitting the 100 mm line into two halves and deembed them from the 200 mm line, the connector and 50 mm of line will be removed on both sides. This will remove the effects caused by the connectors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# deembedding using IEEEP370 impedance corrected 2xthru method\n",
+    "dm = IEEEP370_SE_NZC_2xThru(dummy_2xthru = TL100, name = '2xthru')\n",
+    "side1 = dm.s_side1\n",
+    "side1.name = 'side1'\n",
+    "side2 = dm.s_side2\n",
+    "side2.name = 'side2'\n",
+    "d_dut = dm.deembed(TL200)\n",
+    "d_dut.name = 'd_dut'\n",
+    "\n",
+    "# plot them all\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.suptitle('Connector and launch model')\n",
```

Comment:
Maybe just "connector model"? I was thinking first that there were two networks plotted here.

Reply:
Sounds good !